### ISSUE TITLE
Error: Assertion failed: unable to find node from unique Trail id

### ISSUE THREAD

**Issue by samreid** [phetsims/circuit-construction-kit-common#593]
@jessegreenberg said on slack:

> Well, I am still not sure what the problem is, but it is caused by lines 287 in VertexNode and 226 in CircuitElementNode. Both of these are called in response to up events, which then trigger focusin and focusout events. Is the mirror-inputs wrapper having trouble with nested input like this? I need to move on for a bit, but happy to help out if I can!
>
> Also, I found that clicking in the source sim after load gets the problem to happen right away, since focus related events may not happen if focus is in the dev tools or somewhere else. (edited)

**Comment by samreid** [comment]
@zepumph can we schedule a time to collaborate on this?  It seems like a potential a11y/phet-io overlap.

**Comment by zepumph** [comment]
@samreid and I paired on this today. We found that an easy way to reproduce this bug is to fuzz mirror inputs in CCK dc, and then click randomly  with a mouse/touch in the top sim. Within 10 seconds we could reliably hit this bug.

We found that push/popNonPlayback from dataStream.js solves this error because we saw focus/blur events occurring downstream of other playback events (like mouseDown). Hopefully a general solution will be handled over in https://github.com/phetsims/phet-io/issues/1693. Marking on-hold.
